iceoryx_examples/iceperf/iceperf_leader.cpp

```diff
@@ -80,13 +98,13 @@ void IcePerfLeader::doMeasurement(IcePerfBase& ipcTechnology) noexcept
     std::cout << "#### Measurement Result ####" << std::endl;
     std::cout << m_settings.numberOfSamples << " round trips for each payload." << std::endl;
     std::cout << std::endl;
-    std::cout << "| Payload Size [kB] | Average Latency [µs] |" << std::endl;
+    std::cout << "| Payload Size      | Average Latency [µs] |" << std::endl;
```

Comment:
I have to run and check the output. Currently I somewhat fear that for large payloads it isn't anymore easily distinguishable how large they actually are.

One idea would be to have an output like this
```
16 [B]
32 [B]
...
1 [kB]
2 [kB]
...
1 [MB]
2 [MB]
...
```

What do you think?

Reply:
Yes, would be nice ;)